tests/telemetry/test_payloads.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from ddtrace.internal.telemetry.data.application import get_version
+from ddtrace.internal.telemetry.data.integration import create_integration
+from ddtrace.internal.telemetry.data.metrics import Series
+from ddtrace.internal.telemetry.data.payload import AppClosedPayload
+from ddtrace.internal.telemetry.data.payload import AppGenerateMetricsPayload
+from ddtrace.internal.telemetry.data.payload import AppIntegrationsChangedPayload
+from ddtrace.internal.telemetry.data.payload import AppStartedPayload
+from ddtrace.internal.telemetry.data.payload import Payload
+
+
+def test_app_started_payload():
+    payload = AppStartedPayload()
+
+    assert payload.request_type() == "app-started"
+    assert len(payload.dependencies) > 0
+    assert payload.to_dict() == {"dependencies": payload.dependencies}
```

Comment:
for this we should assert on what is in the dependencies.

do we do we properly create the dependency objects.

Reply:
I did this for the integration changed event too